src/ci/github-actions/ci.yml

```diff
@@ -523,17 +531,14 @@ jobs:
           # This target only needs to support 11.0 and up as nothing else supports the hardware
           - name: dist-aarch64-apple
             env:
-              SCRIPT: ./x.py dist bootstrap --include-default-paths --stage 2
+              SCRIPT: ./x.py dist bootstrap --include-default-paths --host=aarch64-apple-darwin --target=aarch64-apple-darwin
               RUST_CONFIGURE_ARGS: >-
-                --build=x86_64-apple-darwin
-                --host=aarch64-apple-darwin
-                --target=aarch64-apple-darwin
                 --enable-full-tools
                 --enable-sanitizers
                 --enable-profiler
-                --disable-docs
                 --set rust.jemalloc
                 --set llvm.ninja=false
+                --set rust.lto=thin
```

Comment:
Hmm, I'm not sure we should have sneaked LTO in here with the change to M1 builders. Hopefully it won't cause any issues, but next time I'd prefer to do it in a separate PR.